style.css

```diff
@@ -1,22 +1,43 @@
 html, body {
   margin: 0;
   padding: 0;
-  font-family: 'Poppins', sans-serif;
-  font-size: 16px;
+  font-family: 'Varela Round', sans-serif;
+  font-size: 24px;
   background-color: #fff;
   width: 100vw;
   overflow-x: hidden;
-  color: #000;
+  color: #424242;
+}
+
+@media screen and (max-width: 1380px) {
```

Comment:
Why revert the changes that were made by someone else?
Did you do a bad merge conflict resolution?